src/xcode/ENA/ENA/Source/Scenes/Events/Checkin/TraceLocationCheckin/TraceLocationCheckinViewModel.swift

```diff
@@ -25,9 +25,8 @@ final class TraceLocationCheckinViewModel {
 		self.locationAddress = traceLocation.address
 		self.locationDescription = traceLocation.description
 		self.shouldSaveToContactJournal = store.shouldAddCheckinToContactDiaryByDefault
-		// max duration in the picker is 23:45
-		let maxDurationInMinutes = (23 * 60) + 45
-		self.duration = TimeInterval(min(traceLocation.suggestedCheckoutLength, maxDurationInMinutes) * 60)
```

Comment:
Is this no longer necessary or was it moved?

Reply:
The logic was moved to the trace location because otherwise I would have had to duplicate it.

Reply:
Ah thanks I see it now :)